zabbix_metrics_watcher.py

```diff
@@ -85,8 +86,12 @@ def send_zabbix_notification(self, zabbix_host, osbs_master, concurrent_builds):
                 zabbix_result[k] = v
             zabbix_result['upload_size_mb'] = self.upload_size_mb
             try:
-                zabbix_result['pulp_push_speed'] =\
-                    self.upload_size_mb / float(zabbix_result['pulp_push'])
+                if 'pulp_push' in zabbix_result.keys():
+                    zabbix_result['pulp_push_speed'] =\
+                        self.upload_size_mb / float(zabbix_result['pulp_push'])
+                elif 'pulp_sync' in zabbix_result.keys():
```

Comment:
I'm not sure this is correct. The `pulp_sync` timing only reflects the time Pulp takes to sync in from the v2 registry, not the time it takes to push to that registry -- both parts together are the v2 equivalent of the v1 `pulp_push` work.

In any case, I don't think this is very useful for a v2 workflow as it is no longer measuring any single thing. You can't use it to estimate the bandwidth of any particular connection.